src/test/java/org/folio/search/service/IndexServiceTest.java

```diff
@@ -265,6 +270,7 @@ void reindexInventory_positive_recreateIndexIsTrue() {
     verify(indexRepository).dropIndex(indexName);
     verify(consortiumInstanceService).deleteAll();
     verifyNoInteractions(locationService);
+    verifyNoInteractions(campusService);
```

Comment:
Probably not required but location-units are tied to inventory reindexing endpoint & service